config/services/listeners.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <service id="sylius_price_history.listener.channel_pricing_change" class="Sylius\PriceHistoryPlugin\EventListener\ChannelPricingChangeListener">
```

Comment:
No matter how we define services, we should do it consistently, because now some have a suffix `_plugin` and some do not. I would be in favour of not using this suffix, but we should change the configuration tree. Of course, we can do that in a separate PR